web/gds-user-ui/src/modules/dashboard/member/components/MemberTableRow.tsx

```diff
@@ -0,0 +1,42 @@
+import { Td, Tr, Button, HStack, chakra, Tag } from '@chakra-ui/react';
+import { BsEye } from 'react-icons/bs';
+import { formatIsoDate } from 'utils/formate-date';
+import { getVapsNetwork } from '../utils';
+import { VaspType } from '../memberType';
+
+const MemberTableRow: React.FC<{ row: VaspType }> = ({ row }) => {
```

Comment:
Thanks for making this change and for adding the `MemberTableRows` component. Separating the components has made it easier to follow what's happening.